Aniview: Add examples and update params list

s-solodovnikov opened the pull request:
## 🏷 Type of documentation
- [x] new examples

bretg commented on dev-docs/bidders/aniview.md:
```diff
@@ -5,13 +5,17 @@ description: Prebid Aniview Bidder Adapter
 pbjs: true
 biddercode: aniview
 media_types: banner, video
+gpp_supported: true
```
Sorry, this field has been deprecated. As described in https://docs.prebid.org/dev-docs/bidder-adaptor.html, this is now a list of which GPP section IDs are supported. e.g. gpp_sids: tcfeu, tcfca, usnat, usstate_all

bretg commented on dev-docs/bidders/aniview.md:
```diff
@@ -5,13 +5,17 @@ description: Prebid Aniview Bidder Adapter
 safeframes_ok: true
 gvl_id: 780
 sidebarType: 1
+userIds: true
```
This is not a boolean. `userId: (list of supported vendors)`

If you support reading every possible EID, you may say 'all' here.

bretg commented on dev-docs/bidders/aniview.md:
````diff
@@ -21,29 +25,144 @@ For more information about [Aniview Ad Server](https://www.aniview.com/), please
+}];
+```
+
+### Floors
````
this section is not anivew-specific. Is it really necessary?

bretg commented on dev-docs/bidders/aniview.md:
```diff
@@ -21,29 +25,144 @@ For more information about [Aniview Ad Server](https://www.aniview.com/), please
+  userSync: {
+    filterSettings: {
+      all: {
+        bidders: '*',
```
We cannot let you tell publishers to always include all bidders. Our preference would be to drop this whole section. If necessary, you can ask publishers to sync aniview.

bretg commented:
Is this associated with a code PR? I see a change against your adapter was merged last week.

s-solodovnikov commented:
@bretg 
Thank you for the detailed review. I have clarified the data and corrected it according to your recommendations.

This PR is not associated to the last adapter's changes, we just updated our documentation to have it up-to-date.
